src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -431,10 +530,15 @@ protected virtual void Initialize()
                 // take a snapshot so we can detect function additions/removals
                 _directorySnapshot = Directory.EnumerateDirectories(ScriptConfig.RootScriptPath).ToImmutableArray();
 
-                // Scan the function.json early to determine the binding extensions used
+                // Scan the function.json early to determine the requirements.
                 var functionMetadata = ReadFunctionMetadata(ScriptConfig, TraceWriter, _startupLogger, FunctionErrors, _settingsManager);
-                var extensionLoader = new ExtensionLoader(ScriptConfig, TraceWriter, _startupLogger);
-                var usedBindingTypes = extensionLoader.DiscoverBindingTypes(functionMetadata);
+                var usedBindingTypes = DiscoverBindingTypes(functionMetadata);
+
+                bool useLazyLoad = false; // todo - https://github.com/Azure/azure-webjobs-sdk-script/issues/1637
```

Comment:
See my comment [here](https://github.com/Azure/azure-webjobs-sdk-script/pull/1923#discussion_r141694908). 

It's hard to tell because of the method moves, but aside from the feature flag, is there any other functional change? Or just refactoring?